### PR TITLE
(SIMP-5372) add missing faillock system-auth

### PR DESCRIPTION
**Pull request by zach-simp**
Altered system auth_spec tests to match bug fix
Created new acceptance suite to test faillock
Added .fixture.yml requirements for faillock test

SIMP-5372 #close

### CODE REVIEW

**Comment by trevor-vaughan** [comment]
Having trouble running the tests. Debugging

**Comment by trevor-vaughan** [comment]
Need to make a few more adjustments to the test setup

File: spec/acceptance/nodesets/default.yml [hunk 1]
```diff
@@ -24,5 +31,8 @@ HOSTS:
         gpgkeys:
           - https://getfedora.org/static/352C64E5.txt
 CONFIG:
+<% if ENV['BEAKER_PUPPET_COLLECTION'] -%>
+  puppet_environment: <%= ENV['BEAKER_PUPPET_COLLECTION'] %>
```
Comment (lnemsick-simp) [inline spec/acceptance/nodesets/default.yml]: I don't see puppet_environment in any beaker-related gems.  Is this a typo?  Should it be puppet_collection?

Comment (zach-simp) [reply spec/acceptance/nodesets/default.yml]: I have made the requested change to the nodesets to correct puppet_environment to puppet_collection.

**Comment by trevor-vaughan** [comment]
Minor change but a lot of test updates. Everything looks good now.

**Pull request merged**